recipe/fix-cmake.patch

```diff
@@ -0,0 +1,43 @@
+diff --git a/setup.py b/setup.py
+index 05d885b..de47fb9 100644
+--- a/setup.py
++++ b/setup.py
+@@ -38,7 +38,7 @@ class CMakeBuild(build_ext):
+     def build_extension(self, ext):
+         import sysconfig
+ 
+-        import torch
++        # import torch
+ 
+         extdir = os.path.abspath(osp.dirname(self.get_ext_fullpath(ext.name)))
+         self.build_type = "DEBUG" if self.debug else "RELEASE"
+@@ -51,8 +51,8 @@ class CMakeBuild(build_ext):
+         if not osp.exists(self.build_temp):
+             os.makedirs(self.build_temp)
+ 
+-        WITH_CUDA = torch.cuda.is_available()
+-        WITH_CUDA = bool(int(os.getenv('FORCE_CUDA', WITH_CUDA)))
++        # WITH_CUDA = torch.cuda.is_available()
++        WITH_CUDA = bool(int(os.getenv('FORCE_CUDA', False)))
```

Comment:
Leaving a remark for posterity that this was the reason why we had "needed" pytorch in the build environment. Well done in patching that out.